includes/wikia/tests/UserPasswordTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Wikia\Service\Helios\ClientException;
+use Wikia\DependencyInjection\Injector;
+
+class UserPasswordTest extends WikiaBaseTest
+{
+
+	const TEST_USER_ID = 2257663;
+
+	/** @var User */
+	protected $testUser;
+
+	protected static $currentInjector;
+
+	public static function setUpBeforeClass() {
+		self::$currentInjector = Injector::getInjector();
+	}
+
+	public function setUp() {
+		parent::setUp();
+		$this->testUser = User::newFromId( self::TEST_USER_ID );
+	}
+
+	public static function tearDownAfterClass() {
+		Injector::setInjector( self::$currentInjector );
+	}
+
+	public function testShouldSetNewPassword() {
+		$this->assertTrue( $this->testUser->setPassword( "ok" ) );
```

Comment:
These tests are actually calling Helios. I think such integration tests should be covered by Selenium tests, and we should mock here instead as we are relying on the stability of an external service for the stability of MW's tests, which isn't ideal. So, I'd suggest mocking our contract with Helios here.